server/routes/induction/common/workInterestRolesView.ts

```diff
@@ -7,22 +7,19 @@ export default class WorkInterestsRoleView {
     private readonly backLinkUrl: string,
     private readonly backLinkAriaText: string,
     private readonly workInterestRolesForm: WorkInterestRolesForm,
-    private readonly errors?: Array<Record<string, string>>,
```

Comment:
Because we dont validate this form, there will never be errors, so we can remove the errors property from this view class